core/src/memory/binary_page_memory_resource_impl.cpp

```diff
@@ -14,24 +14,43 @@
 // System include(s).
 #include <algorithm>
 #include <memory>
-#include <stack>
+#include <optional>
 #include <stdexcept>
 
+#ifdef _MSC_VER
```

Comment:
I would modify this to using `VECMEM_HAVE_LZCNT_U64` as well. Since that test explicitly checks the existence of this header file as well...

Reply:
Fixed!